### DISCUSSION TITLE
Import from Tenant A and Export to Tenant B.

### DISCUSSION THREAD

**Discussion by Bono-Boy** [Micke-K/IntuneManagement#121] [Q&A] [answered]
First of all. Thank you very much for this great product.
I created a backup of all settings from tenant A and would like to import it into tenant B.
Are there things that I need to watch for? What does Name filter do?
![image](https://user-images.githubusercontent.com/95429653/224027829-a0c679a9-da64-4f24-bec4-6603caad454f.png)

**Comment by Micke-K** [comment, marked as answer]
Hello!

Thank you for using the tool!

Name filter is s way to limit the policies to import eg if you have production and test policies in the same tenant. 

If you want to import everything then just leave it empty. 

Cheers!

**Reply by Bono-Boy** [reply]
Thank you for the swift response. Much appreciated.
